.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
```

Comment:
I think it is better to release on tags, not on push to main

Reply:
https://github.com/Near-One/aurora-eth-connector/blob/e19b8ef6ce835012aa146a3815937ec0e7d8fdc3/.github/workflows/update-contract.yaml#L3